network-api/networkapi/wagtailpages/migrations/0051_buyersguideproductpagearticlepagerelation.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.13 on 2022-08-23 17:25
```

Comment:
This migration creates the relationship between Product Pages and Related Articles